src/Modules/Build.lua

```diff
@@ -156,11 +156,57 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		end
 	end
 	self.controls.pointDisplay.width = function(control)
-		local used, ascUsed = self.spec:CountAllocNodes()
-		local usedMax = 99 + 22 + (self.calcsTab.mainOutput.ExtraPoints or 0)
-		local ascMax = 8
-		control.str = string.format("%s%3d / %3d   %s%d / %d", used > usedMax and "^1" or "^7", used, usedMax, ascUsed > ascMax and "^1" or "^7", ascUsed, ascMax)
-		control.req = "Required level: "..m_max(1, (100 + used - usedMax))
+		if (self.calcsTab.mainOutput.ExtraPoints == nil) 
+		then 
+		  bandit = 0 
+		else 
+		  bandit = self.calcsTab.mainOutput.ExtraPoints or 0 
+		end
```

Comment:
This section is unnecessary.  Lua treats `nil` as false, so you can just replace `bandit` with `self.calcsTab.mainOutput.ExtraPoints or 0`
```suggestion
```

Reply:
i did this small section because #2640 referenced to this codeline - i guess i can remove it then...

Reply:
Sounds like it was fixed for them in 2.3.0...somehow.  In Lua, line 163 acts as a ternary of sorts, so the extra if/else isn't needed.  See this page for more examples: https://www.lua.org/pil/3.3.html